google_functions_v2/sensors_db_fake.py

```diff
@@ -27,8 +28,16 @@ def read_starting_from(self, date: dt.datetime) -> t.Tuple[t.List[sen.Sensor], t
         sensors: t.List[sen.Sensor] = list(name_to_sensor_temp.values())
         return sensors, messages
 
+    def read_starting_from(self, date: dt.datetime) -> t.Tuple[t.List[sen.Sensor], t.Set[str]]:
+        return self.__get_sensors_from(date)
+
 
     def delete_before(self, date: dt.datetime) -> None:
         del_before_ind: int = bs.bisect(self.data, date, key=lambda dd: dd.time)
         del self.data[:del_before_ind]
 
+    def read_last_result(self) -> t.Tuple[t.List[sen.Sensor], t.Set[str]]:
+        last_result = self.data[-1]
```

Comment:
Does not look right...

* Why do you run `bisect` (in `__get_sensors_from`) if you already know that it is the last one?
* Tests are missing
* Entire thing relies on the fact that `write` gets data in sorted by timestamp order. May be add an assert that checks that it is true?